hudi-hadoop-common/src/test/java/org/apache/hudi/common/fs/TestStorageSchemes.java

```diff
@@ -39,6 +39,10 @@ public void testStorageSchemes() {
     assertTrue(StorageSchemes.isSchemeSupported("afs"));
     assertFalse(StorageSchemes.isSchemeSupported("s2"));
 
+    assertTrue(StorageSchemes.isConditionalWriteSupported("s3"));
+    assertTrue(StorageSchemes.isConditionalWriteSupported("s3a"));
+    assertTrue(StorageSchemes.isConditionalWriteSupported("gs"));
+
```

Comment:
Add a `assertFalse` on one of the file storage scheme that does not support conditional writes?